pkg/provision/providers/qemu/launch.go

```diff
@@ -448,7 +448,7 @@ func launchVM(config *LaunchConfig) error {
 			)
 		case config.USBPath != "":
 			args = append(args,
-				"-drive", fmt.Sprintf("if=none,id=stick,format=raw,file=%s", config.USBPath),
+				"-drive", fmt.Sprintf("if=none,id=stick,format=raw,read-only=on,file=%s", config.USBPath),
```

Comment:
make QEMU USB stick emulation read-only, so that we can mount same file to multiple VMs